### DISCUSSION TITLE
HTML full support

### DISCUSSION THREAD

**Discussion by wmdhosting** [craftcms/ckeditor#176] [Q&A] [answered]
This new CK version have full HTML support.. as much i have seen

How to enable that so it works .. as still  all classes and custom code is removed..

**Comment by brandonkelly** [comment, marked as answer]
You can tell CKEditor to allow custom HTML/classes by configuring its [General HTML Support](https://ckeditor.com/docs/ckeditor5/latest/features/html/general-html-support.html#configuration) feature
